pkgs/applications/misc/wmenu/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, fetchpatch
+, pkg-config
+, cmake
```

Comment:
you can also remove `cmake` here, and also there is still an erroneous commit to "argocd-autopilot" in this PR,
if you got to "Commits" tab of this PR, you can see it there.